.github/renovate.json

```
@@ -0,0 +1,4 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": ["github>aladdin-add/eslint-gha-configs//.github/renovate.json5"]
```

It may be worth just using the same config that is in eslint :thinking:

```suggestion
    "extends": ["github>eslint/eslint//.github/renovate.json5"]
```